src/components/RoomView/Board.jsx

```diff
@@ -34,10 +34,12 @@ const Board = () => {
     <div style={style}>
       {words.wordsInGame.map((singleWord) => {
         return (
-          <>
+   <div key={singleWord.id}>
+
             <Card singleWord={singleWord} />
-            <div style={{ display: "none" }}>Reveal Team</div>
-          </>
+  {/* maybe set this to visible after players select it?  */}
```

Comment:
For this PR i think anything is fine for now. We haven't thought that much the exact logic for how the card's associated team/color will be revealed yet. so id say dont worry about it now